src/opening-period/time-span/TimeSpan.scss

```diff
@@ -37,3 +38,13 @@
   justify-content: space-between;
   align-items: flex-end;
 }
+
+.checkbox-container {
+  display: flex;
+  align-items: center;
+  height: var(--spacing-3-xl);
```

Comment:
Could this be in pixels with a comment? // 56px is the input height etc.

Reply:
- [ ] Add comment about pixel value and reasoning